Src/BootCamp.Chapter/ContactsCenter.cs

```diff
@@ -1,27 +1,66 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public class ContactsCenter
     {
-        private readonly List<Person> _people;
+         private readonly string _peopleFile;
 
         public ContactsCenter(string peopleFile)
         {
-            // load people
+            if (!File.Exists(peopleFile))
+            {
+                throw new FileNotFoundException("File is made up"); 
+            }
+                      
+            _peopleFile = File.ReadAllText(peopleFile); 
+           
+            if (String.IsNullOrWhiteSpace(_peopleFile))
+            {
+                throw new Exception(); 
+            }
+         
         }
 
+       
         /// <summary>
         /// Gets people by filter predicate.
         /// </summary>
         /// <returns></returns>
         public List<Person> Filter(Predicate<Person> predicate)
         {
             var people = new List<Person>();
-            // ToDo: implement applying filter.
+            var splittedList = _peopleFile.Split($"{Environment.NewLine}");
+            for (int i = 1; i < splittedList.Length -1 ; i++)
```

Comment:
Comment that you've skipped the header row - if i was just reading this code i'd not understand why you did this. In fact i had to double check the data file to see if you did this because of a header or not :D

Reply:
done